pose_to_segments/bin.py

```diff
@@ -62,26 +63,30 @@ def get_args():
     parser.add_argument('-i', required=True, type=str, help='path to input pose file')
     parser.add_argument('-o', required=True, type=str, help='path to output elan file')
     parser.add_argument('--video', default=None, required=False, type=str, help='path to video file')
+    parser.add_argument('--subtitle', default=None, required=False, type=str, help='path to subtitle file')
+    parser.add_argument('--model', default='model_E1s-1.pth', required=False, type=str, help='path to model file')
 
     return parser.parse_args()
 
-
 def main():
     args = get_args()
 
     print('Loading pose ...')
     with open(args.i, "rb") as f:
-        pose = process_pose(Pose.read(f.read()))
-
+        if 'E4' in args.model:
+            pose = process_pose(Pose.read(f.read()), optical_flow=True, hand_normalization=True)
+        else:
+            pose = process_pose(Pose.read(f.read()))
+        
     print('Loading model ...')
     install_dir = os.path.dirname(os.path.abspath(__file__))
-    model = load_model(os.path.join(install_dir, "dist", "model_E1s-1.pth"))
+    model = load_model(os.path.join(install_dir, "dist", args.model))
 
     print('Estimating segments ...')
     probs = predict(model, pose)
 
-    sign_segments = probs_to_segments(probs["sign"])
-    sentence_segments = probs_to_segments(probs["sentence"], .8, .8)
+    sign_segments = probs_to_segments(probs["sign"], 60, 50)
+    sentence_segments = probs_to_segments(probs["sentence"], 85, 80)
```

Comment:
paper says 90, 90 for this step

Reply:
Sure, let's also use this on SignTube then, I set `85, 80` to align with SignTube.